repos/spack_repo/builtin/packages/py_bidskit/package.py

```diff
@@ -15,18 +15,28 @@ class PyBidskit(PythonPackage):
 
     license("MIT")
 
+    version("2025.1.30", sha256="91daa1041b07b029da2e720b91270be9ab39c42ccbb6e8d8bb302441ea1a1bf5")
     version("2023.9.7", sha256="029d9aecbbcb2df733858ceb3e6d5dd5013c36e431e40fb522a580adc7b667a5")
     version("2023.2.16", sha256="b2e4e3246d43a6f00af6c0391ec8fecc59405241de1ea9ca68eb4d8128d62c7b")
     version(
         "2022.10.13", sha256="576b92cef187032c73f64e2e6a5b0be0c06771442048a33c55e224b3df0aae3a"
     )
 
+    depends_on("python@3.10:3", type=("build", "run"), when="@2024.11.7:")
     depends_on("python@3.7:3", type=("build", "run"))
+
+    # requirements.txt
+    # ignore upper version limits, because it works with newer versions as well
+    depends_on("py-setuptools@72.1:", type=("build", "run"), when="@2025.1.30:")
     depends_on("py-setuptools", type="build")
 
+    depends_on("py-pydicom@2.3:", type=("build", "run"), when="@2025.1.30:")
```

Comment:
Shouldn't this be `@2.3.1:`?

Reply:
Yes, you are right. This is fixed now. Thanks.